### PR TITLE
Improve Accessibility Features

### PR DESCRIPTION
**Pull request by DonaldRG**
## Description of changes
- Add support for accessibility.
- Add dynamic font support by default.

[CSDK-608]: https://verygoodsecurity.atlassian.net/browse/CSDK-608?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSExpDateTextField.swift [hunk 1]
```diff
@@ -72,6 +72,39 @@ public final class VGSExpDateTextField: VGSTextField {
       // default behaviour in case field setup with VGSConfiguration
       setupDatePicker()
     }
+    
+    override func updateAccessibilityValues() {
```
[the remark this answers is not in the document]
Comment (DonaldRG) [reply Sources/VGSCollectSDK/UIElements/Text Field/VGSExpDateTextField.swift:76]: As we talked I will remove it 👍

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift [hunk 2]
```diff
@@ -93,19 +93,58 @@ public class VGSTextField: UIView {
...
+        get {
+            return textField.accessibilityTraits
+        }
+        set {
+            textField.accessibilityTraits = newValue
+        }
+    }
+    
+    /// Localized string that represents the value of the element
+    public var textFieldAccessibilityValue: String? {
```
Comment (dmytrokhl) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift:140]: should it return input value?

Comment (DonaldRG) [reply Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift:140]: You are right, I think it is better to return nil

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift [hunk 3]
```diff
@@ -93,19 +93,58 @@ public class VGSTextField: UIView {
...
+            return textField.isAccessibilityElement
+        }
+        set {
+            textField.isAccessibilityElement = newValue
+        }
+    }
+    
+    /// A collection of accessibility trait masks that best describes the characterize of the element
+    /// See `UIAccessibilityConstants.h` for a list of possible traits.
+    public var textFieldAccessibilityTraits: UIAccessibilityTraits {
```
Comment (dmytrokhl) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift:130]: It seems not necessary since it's always textField

Comment (DonaldRG) [reply Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift:130]: Yes, I agree

**Review by dmytrokhl — APPROVED** [review]

**Pull request merged**